catalog/transformer/catalog-transformer-service-atom/src/main/java/ddf/catalog/transformer/response/query/atom/AtomTransformer.java

```diff
@@ -233,6 +235,35 @@ public BinaryContent transform(SourceResponse sourceResponse, Map<String, Serial
       startIndex.setText(Integer.toString(sourceResponse.getRequest().getQuery().getStartIndex()));
     }
 
+    if (getCount(sourceResponse) != 0) {
+      addResultElements(sourceResponse, currentDate, feed);
+    }
+
+    ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+    try {
+
+      tccl = Thread.currentThread().getContextClassLoader();
```

Comment:
✏️  Should be moved outside the first `try`, that way you wouldn't need to embed a second `try/finally` and instead, you could just move the `finally` to the one and only `try` right after the `catch`

Reply:
Looks like this was a preexisting condition, but definitely an easy clean-up

Reply:
cleaned up